public/index.html

```diff
@@ -47,7 +47,7 @@
       }
     </style>
   </head>
-  <body style="background: #1B1B1B; margin:0;">
+  <body style="background: #1B1B1B; margin:0; position: relative; min-height: 100vh;">
```

Comment:
skoro masz już znacznik style to czemu nie wrzucisz tego z body do style? Będzie ładniej wyglądać